### PR TITLE
Fake authentication

### PR DESCRIPTION
**Pull request by forevermatt**
### Added
- Add authentication, but fake it for the moment

### CODE REVIEW

File: src/i18n/index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import phrases from './phrases.json'
 export default function t(key) {
   const phrase = phrases[key]
   if (phrase === undefined) {
+    console.log(key)
```
Comment (captaincrazybro) [inline src/i18n/index.js:6]: Is this a debug `console.log()`?

File: src/pages/index.svelte [hunk 2]
```diff
@@ -1,22 +1,5 @@
...
-import { logout } from '../authn'
-import { GET } from '../data'
-import { onMount } from 'svelte'
-import user from '../authn/user'
+import { goto } from '@roxi/routify'
 
-onMount(async () => {
-  await GET('bearer')  // this helps simulate a page accessed that may require the user be authenticated
-})
+$goto('/home')
```
Comment (captaincrazybro) [inline src/pages/index.svelte:4]: I thought `index.svelte` was for logging in?

**Review by captaincrazybro — APPROVED** [review]

**Review by hobbitronics — APPROVED** [review]

File: src/i18n/index.js (hunk 1 above)
Comment (forevermatt) [reply src/i18n/index.js:6]: Only sort of. If a key is provided that isn't in our phrases file, we don't have any clues as to what it was. This lets us at least see in the console what it was, so we can know what to add.

File: src/pages/index.svelte (hunk 2 above)
Comment (forevermatt) [reply src/pages/index.svelte:4]: It was going to be... but I realized that when I refresh the page on any other route, we don't do the login for unauthenticated users. So my file move was mostly unnecessary after all. (Sorry again.)

**Pull request merged**